server/gameconnection.py

```diff
@@ -92,7 +90,7 @@ def send_message(self, message):
         elif state == PlayerState.JOINING:
             pass
         else:
-            self._logger.exception("Unknown PlayerState")
+            self._logger.exception("Unknown PlayerState: %s", state)
```

Comment:
So here the server expects JOINING or HOSTING... but receives Initializing instead. that's interesting.

Reply:
I think this was a different thing because the `Initializing` thing is a `GameState` not a `PlayerState`